### ISSUE TITLE
Add missing Image options 

### ISSUE THREAD

**Issue by hypest** [wordpress-mobile/gutenberg-mobile#1263]
Let's add the image options available in Aztec but missing in Gutenberg mobile:

- [x] (Full screen) Preview
- [x] Size
- [x] Link To (including "open in new window")
- [x] Alignment

**Comment by cameronvoell** [comment]
So after some research on Image options for Aztec vs Gutenberg across Web, Android, and iOS, I think this issue could be used as a parent issue (or overarching as @hypest [mentioned](https://github.com/wordpress-mobile/gutenberg-mobile/issues/198#issuecomment-519973325) ) for a few sub-tasks.

1. Design Feedback on image options UI in gutenberg-mobile image options menu.
2. (Full screen) Preview
3. Size
4. Link To (including "open in new window")
5. Alignment - already tracked here: #198 

The design feedback task would probably be useful to get going in parallel to me getting started. Currently the 4 new image option features vary greatly between Aztec Android and Aztec iOS. 
**See Aztec Editor screen shots below:**
![image](https://user-images.githubusercontent.com/1103838/62897101-5fbaa900-bd07-11e9-8205-a27b42d2d11a.png)

As a starting point I mocked up how we might include the image option UI selectors in the current gutenberg-mobile image options menu. While we are deciding the best UI for modifying each of these 4 properties in gutenberg-mobile, I can get started on the code that executes the image preview and updates the post content based on how the user updates the image options. 
**Gutenberg-mobile image options starting point mock up:**
![image](https://user-images.githubusercontent.com/1103838/62898331-115ad980-bd0a-11e9-9231-f90c989f4880.png)

If this all sounds good, I'll create new github issues for 1-4, message @iamthomasbishop about feedback on the first design feedback issue, and then get started on 2 through 5. Once I have all the Github issues straightened out, I can create a post in Hogwarts p2 tracking progress on these as well.

**Comment by hypest** [comment]
Will remove this ticket from the project board since the child tickets are already on it.

**Comment by SergioEstevao** [comment]
I'm closing this issue because all the child tickets are already solved and merged.
